src/components/Button/styled/button.js

```diff
@@ -98,7 +98,7 @@ const StyledButton = styled.button.attrs(props => {
     ${props =>
         props.variant === 'neutral' &&
         `
-            background-color: ${COLOR_WHITE};
+            background-color: transparent;
```

Comment:
This variant is not meant to be transparent if you have a good reason for what this needs to be changed then discuss it with @TahimiLeonBravo